src/components/file-component/FileComponent.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+
+import AppButton from '~/components/app-button/AppButton'
+
+import { ButtonVariantEnum, File } from '~/types'
+import { getFormatedDate, spliceSx } from '~/utils/helper-functions'
+import { styles } from '~/components/file-component/FileComponent.styles'
+
+interface FileComponentProps {
+  file: File
+}
+
+const openFile = (file: File) => {
+  window.open(file.url, '_blank', 'noopener noreferrer')?.focus()
+}
```

Comment:
I don't understand why there needs focus.

And you use the same in two files. Move out to one file and don't repeat yourself.

Reply:
> I don't understand why there needs focus.
> 
> And you use the same in two files. Move out to one file and don't repeat yourself.

removed focus